doc/cprover-manual/contracts-history-variables.md

```diff
@@ -0,0 +1,36 @@
+# History Variables
+
+### Syntax
+
+```c
+__CPROVER_old(*identifier*)
+```
+
+Refers to the value of a given object in the pre-state of a function within the
+_ensures_ clause.
+
+**Important.** This predicate may be used only within the context of `__CPROVER_ensures`.
+
+### Parameters
+
+`__CPROVER_old` takes a single argument, which is the identifier corresponding to
+an input to the function. For now, only inputs of scalar, structs, or pointer types are supported.
+
```

Comment:
"input" of the function is ambiguous. You want to say "parameter"

Reply:
Fixed it.